azure-keyvault/azure/keyvault/models/json_web_key.py

```diff
@@ -58,14 +67,17 @@ class JsonWebKey(Model):
         'qi': {'key': 'qi', 'type': 'base64'},
         'p': {'key': 'p', 'type': 'base64'},
         'q': {'key': 'q', 'type': 'base64'},
+        'x': {'key': 'x', 'type': 'base64'},
+        'y': {'key': 'y', 'type': 'base64'},
         'k': {'key': 'k', 'type': 'base64'},
         't': {'key': 'key_hsm', 'type': 'base64'},
     }
 
-    def __init__(self, kid=None, kty=None, key_ops=None, n=None, e=None, d=None, dp=None, dq=None, qi=None, p=None, q=None, k=None, t=None):
+    def __init__(self, kid=None, kty=None, key_ops=None, crv=None, n=None, e=None, d=None, dp=None, dq=None, qi=None, p=None, q=None, x=None, y=None, k=None, t=None):
```

Comment:
This is a backwards breaking change b/c of the ordering of the params in the rest spec.  I will update the rest spec and regenerate